controllers/getcoursescontroller.go

```diff
@@ -31,23 +31,17 @@ type dataTokensAndCurrencies struct {
 
 // make Response for get prices
 type prices struct {
-	Currency      string              `json:"currency"`
-	Rates         []percentChanges `json:"rates"`
-	//PercentChange string              `json:"percent_change,omitempty"`
-}
-
-type percentChanges struct {
-	Rates map[string]string
-	PercentChange string `json:"percent_change,omitempty"`
+	Currency string              `json:"currency"`
+	Rates    []map[string]string `json:"rates"`
 }
 
 // make Response list API
 type listApi struct {
```

Comment:
Is there a method to this structure? If not, the use of structure is unnecessary

Reply:
Are you sure?)